src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -67,6 +67,14 @@ public void ValidateAgainstBlobs(object obj, string[] blobs)
         }
 
         [Fact]
+        public void UnitySerializationHolderWithAssemblySingleton()
```

Comment:
I want to move this test to System.Runtime.Tests but I will wait with that until I commited my change into the release/2.0.0 branch because it requires the bigger change https://github.com/dotnet/corefx/pull/23940 to handle "blobs" outside of S.R.Serialization.Formatters.